applications/Web3Box.md

```diff
@@ -0,0 +1,142 @@
+# W3F Open Grant Proposal
+
+- **Project Name:** Web3Box
+- **Team Name:** Web3Box Labs
+- **Payment Address:** 0x2755213c8435F15D9929101357686a9C1Bec280A (USDC)
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 1
+
+## Project Overview
+
+### Overview
+
+**Web3Box is committed to creating a Web3 UGA platform with a secure environment worldwide**
+
+- In order to prevent inevitable frauds, we propose a privacy-safe sandbox concept, Web3Box, so that all the information of Polkadot ecosystem users is controlled in their local trusted environment.
+- Web3Box will become the Web3 portal for Polkadot ecosystem users, giving them one-stop access to all Web3 applications. Polkadot ecosystem users can manage their multi-chain assets and track asset changes, ensuring asset security.
+- In response to the frequent Web3 asset security incidents,  Polkadot ecosystem users can track the risks of Polkadot ecosystem projects through the Web3Box, so as to choose Web3 portfolio safely.
+
+
+
+
+### Project Details
+
+#### Web3Box Architecture ####
+**![](https://lh3.googleusercontent.com/tlealryMkOZD0i7dK9LcU_TTsbO5J_13YcgeOl8hWI56sLVtnyYEDDY-dCeYLVa7_i4k4PIdHCkNqTjA5BFtqYKBJJMUh5jkklAxK9-9PKljwMfoq2a0l-WoMXgTRToE5Pih0m2qGXUoi_Yjl7dSNIo3J_3uAGBW4qaVxN3TUGKjQ88VxuhntBRcddz6JA)**
+
+
+
+
+#### Key Modules ####
+1. Privacy-safe sandbox
+- Relay chains & Parachains Assets Management
+-  Relay chains & Parachains Assets Portfolio
+
+2. Risk assessment
+- Parachains Projects Scoring
+- Parechains Projects Risk Report
+- Parachains Abnormal warning
+
+3. Web3 Tool/Feed aggregator
+ - Web3 tool navigation
+ - Web3 content aggregator
+ - Web3 information source customization
+
+#### Web3Box VS Browser ####
+![](https://lh3.googleusercontent.com/x5iDstqgO_-ljPxqvNfHR07JF0Bo8xl1aiBEVKxOfxNqXrydZW2drKlkQQU5oiviknzsgqvGGD90k-UlT4Y5TbZn9c4f7-pOfIanXLSuXeb_svjQOMrMcIGddV2VY5m6MohVoLn2rwBNW85QcHZyYy7GMqWVu-xKt_QbRJ6gQCj4Rg1mlISImyXhd3fsaw)
```

Comment:
What does it mean that "Rights" are "de-centralised"?

Reply:
Web3Box is open source and will provide user-friendly customized plug-in features in the future, which are related to the Web3 Tool/Feed aggregation module.